scripts/pre_commit_linter.py

```diff
@@ -44,12 +44,11 @@
 
 __author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
 
+import sys
+import time
 import argparse
 import os
 import subprocess
-import sys
-import time
```

Comment:
will do! this was actually throwing off some pylint warnings, perhaps we can talk about which rules we want to ignore?
/cc @seanlip